src/rqa/rqa.jl

```diff
@@ -349,8 +347,7 @@ of recurrence times [1].
 [DOI:10.1103/physreve.75.036222](https://doi.org/10.1103/physreve.75.036222)
 
 """
-nmprt(R::ARM; kwargs) = maximum(verticalhistograms(R; kwargs...)[2])
-
+nmprt(R::Union{ARM,SparseMatrixCSC}, kwargs...) = maximum(verticalhistograms(R.data;theiler=deftheiler(R), kwargs...)[2])
```

Comment:
I don't understand. How can this definition work? A `SparseMatrixCSC` does not have a field `.data`.

Reply:
That should only be `R`, but your point does still stand. I'm not sure. But I did change it to simply just `R` so the right type does get fed through. 

Must be something internally with SparseMatrixCSC falling back on something? I have no idea.